packages/plugin-ext-metrics/.gitignore

```diff
@@ -0,0 +1,25 @@
+.DS_Store
```

Comment:
I'm unsure this .gitignore file is required there ? as we have global one https://github.com/eclipse-theia/theia/blob/master/.gitignore
it looks like a dup